docs/src/literate-tutorials/reactive_surface.jl

```diff
@@ -0,0 +1,239 @@
+# # [Reactive Surface](@id tutorial-reactive-surface)
+#
+# ![](reactive_surface.gif)
+#
+# *Figure 1*: Reactant concentration field of the Gray-Scott model on the unit sphere.
+#
+#-
+#md # !!! tip
+#md #     This example is also available as a Jupyter notebook:
+#md #     [`reactive_surface.ipynb`](@__NBVIEWER_ROOT_URL__/examples/reactive_surface.ipynb).
+#-
+#
+# ## Introduction
+#
+# This tutorial gives a quick tutorial on how to assemble and solve time-dependent problems
+# on embedded surfaces.
+#
+# For this showcase we use the well known Gray-Scott model, which is a well-known reaction-diffusion
+# system to study pattern formation. The strong form is given by
+#
+# ```math
+#  \partial_t r_1 = \nabla \cdot (D_1 \nabla r_1) - r₁*r₂^2 + F *(1 - r₁) \quad \textbf{x} \in \Omega,
+#  \partial_t r_2 = \nabla \cdot (D_2 \nabla r_2) + r₁*r₂^2 - r₂*(F + k ) \quad \textbf{x} \in \Omega,
```

Comment:
I think you need `\begin{aligned}` or something around these, see https://ferrite-fem.github.io/Ferrite.jl/previews/PR715/tutorials/reactive_surface/#Introduction